client/src/components/App/Content/UsersManagement/useUsersManagement.ts

```diff
@@ -82,6 +93,157 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort }: useUsersManagem
 
     }
 
+    const fetchSourcesOrganization = () => {
+        logger.info({
+            service: Service.CLIENT,
+            severity: Severity.LOW,
+            workflow: 'Fetching sourcesOrganization',
+            step: 'launching sourcesOrganization request',
+            user: user.id,
+            investigation: epidemiologyNumber
+        })
+        axios.get('/users/sourcesOrganization')
+            .then(result => {
+                result?.data && setSourcesOrganization(result.data);
+                logger.info({
```

Comment:
You dont want to print a sucess log if result.data isnt correct. In addition you also should check result.headers['content-type'].includes('application/json') as in exposureForm line 96, in order to prevent crash due to schema

Reply:
same for all requests returning arrays

Reply:
fixed